### PR TITLE
Trying to add a space into a file

### PR DESCRIPTION
**Pull request by shouldDoFine**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/LasTshaMAN/Threads/pull/3" target="_blank">ReviewNinja

File: src/test/java/ru/sbt/threads/task/TaskPerformanceTest.java [hunk 1]
```diff
@@ -42,6 +42,7 @@ public void setUp() throws Exception {
         executor = Executors.newFixedThreadPool(5);
 
         this.tasks = new ArrayList<>();
+
```
Comment (LasTshaMAN) [inline src/test/java/ru/sbt/threads/task/TaskPerformanceTest.java]: some comment

Comment (LasTshaMAN) [inline src/test/java/ru/sbt/threads/task/TaskPerformanceTest.java]: !fix

Comment (LasTshaMAN) [inline src/test/java/ru/sbt/threads/task/TaskPerformanceTest.java]: !resolved

**Pull request closed without merging**